### PR TITLE
Med req r4 synonym

### PR DESCRIPTION
**Pull request by UjwalaBheema**
Description
----
MedicationRequest - Expose Order Synonym Id in the MedicationRequest.medicationCodeableConcept
Screenshots before:

Screenshot 2020-08-06 at 11 52 06 AM
Screenshot 2020-08-06 at 11 52 53 AM
Screenshot 2020-08-06 at 11 53 12 AM

Screenshots After changes:
Screenshot 2020-08-03 at 6 35 59 PM
Screenshot 2020-08-03 at 6 37 15 PM
Screenshot 2020-08-03 at 6 28 35 PM

PR Checklist

----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: lib/resources/example_json/r4_examples_medication_request.rb [hunk 1]
```diff
@@ -40,6 +40,14 @@ module Resources
             "ingredient": [
               {
                 "itemCodeableConcept": {
+                  "coding": [
+                    {
+                      "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/synonym",
```
Comment (BS062721) [inline lib/resources/example_json/r4_examples_medication_request.rb:45]: These examples now should point to new sandbox tenant, ref: https://github.com/cerner/fhir.cerner.com/commit/bc0d9d61416173f030bc6af11e263a4830fb1c3f

File: lib/resources/r4/medication_request.yaml [hunk 2]
```diff
@@ -131,6 +137,9 @@ fields:
     - display: RxNorm
       system: https://www.nlm.nih.gov/research/umls/rxnorm
       info_link: https://www.nlm.nih.gov/research/umls/rxnorm/index.html
+    - display: Cerner Medication Request Synonym's for orders
+      system: https://fhir.cerner.com/synonym
```
Comment (UjwalaBheema) [inline lib/resources/r4/medication_request.yaml:141]: @kathypickering  
Display: Cerner Medication Request Synonym's for orders
Fine for the synonym's description

Screenshot 2020-08-20 at 10 39 55 PM

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 1 above)
Comment (UjwalaBheema) [reply lib/resources/example_json/r4_examples_medication_request.rb]: Addressed

File: content/millennium/r4/proprietary-codes-and-systems.md [hunk 3]
```diff
@@ -759,3 +760,14 @@ This system is the category and is only for a pharmacy charge-only order. The co
       "code": "charge-only",
       "display": "Charge-Only"
     }
+
+##### Medication Request Synonym
+
+This system is the synonym id for a order and the ingredients.
```
[the remark this answers is not in the document]
Comment (UjwalaBheema) [reply content/millennium/r4/proprietary-codes-and-systems.md]: Addressed

File: lib/resources/r4/medication_request.yaml (hunk 2 above)
Comment (kathypickering) [inline lib/resources/r4/medication_request.yaml:141]: Change the Details to Details: Millennium Medication Order Synonym
The System should be System:https://fhir.cerner.com/<EHR source id>/synonym

What is Details currently linking to?

Comment (UjwalaBheema) [reply lib/resources/r4/medication_request.yaml]: Addressed changed the system and details 
Screenshot 2020-08-24 at 11 47 56 AM

**Review by aki-mathur — APPROVED** [review]
Looks good to me.

**Pull request merged**